packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -20,6 +20,13 @@ The Devices API supports the following **Authorization Schemes**:
 * SSWS - [API tokens](/docs/reference/core-okta-api/#authentication)
 * Bearer - [OAuth2.0 and OpenID Connect](/docs/concepts/oauth-openid/)
 
+> **Note:** For devices to enroll in Okta and show up in the Devices API, the following actions are required:
+> 1. Admins - Enable Okta FastPass. See [enable FastPass](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/devices/fp/fp-enable.htm)
+> 2. End users with existing mobile Okta Verify enrollments - Migrate existing Okta Verify enrollments to use [FastPass](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/devices/fp/fp-main.htm) after the org migrates to Okta Identity Engine.
+
+> End users with a new enrollment in Okta Verify on an Okta Identity Engine org will have a device record created in the device inventory by default.
```

Comment:
```suggestion
> End users with a new enrollment in Okta Verify on an Okta Identity Engine org have a record created in the device inventory by default.
```

Reply:
End users with a new enrollment in Okta Verify on an Okta Identity Engine org have a device record created in the device inventory by default.

Reply:
@brentschaus-okta I updated the text! Thanks for catching this